_episodes_rmd/06-tidy-data.Rmd

```diff
@@ -0,0 +1,73 @@
+---
+title: "Tidying & packaging datasets"
+teaching: 30
+exercises: 10
+questions:
+- "Which different forms can one dataset have?"
+- "Which advantages and disadvantages do these forms have?"
+- "Which features make a dataset more or less reusable?"
+- "How can we add datasets to R packages?"
+objectives:
+- "Use `tidyr::gather()` to convert wide data to its long form."
+keypoints:
+- "Excel & co. incentivise the wide data format, which may spread variables across columns."
+source: Rmd
+---
+
+```{r, include = FALSE}
+source("../bin/chunk-options.R")
+knitr_fig_path("06-tidy-data-R-")
+```
+
+## Tidying our inflammation datasets
+
+In the preparations episode, we read in, visualised, centered and rescaled a
+dataset about inflammation in 60 patients measured over 40 days. Let's pretend
+we need to tidy it up for further analysis and publication.
+
+```{r inflammation-wide}
+dat <- read.csv(file = "data/inflammation-01.csv", header = FALSE)
+colnames(dat); rownames(dat)
+```
+
+You probably noticed that neither "patients" nor "days" is listed anywhere. Thus,
+we should label the data first, starting with pseudonyms for the patients. If
+this were real data from real human patients, we would have acquired the written
+and informed consent to the pseudonymous data publication right in the beginning
+of the study, during approval by the ethics committee of our study.
+
+```{r inflammation-label}
+patient_ID <- paste("patient", sep = "_", seq(60))
+dat_labelled <- cbind(dat, patient_ID)
```

Comment:
since we use dplyr/tidyr, why not also use `bind_cols` instead of `cbind`?

Reply:
I didn't know about that, thanks for the hint! In this case, though: `Error in cbind_all(x) : Argument 2 must have names`. So, I think we can either stick to `base::cbind` or explain `names()`. What do you think?